src/resources/views/corporation/starbase/ajax/silos-tab.blade.php

```diff
@@ -0,0 +1,153 @@
+  <div class="row">
+    <div class="col-md-12">
+
+              <table class="table datatable compact table-condensed table-hover table-responsive dataTable no-footer" id="DataTables_Table_1">
+                <thead>
+                <tr>
+                    <!-- @todo - setup new words in language files -->
+                  <th>{{ trans_choice('web::seat.type', 1) }}</th>
+                  <th class="sorting" tabindex="0" aria-controls="silo_table" rowspan="1" colspan="1" aria-label="Location: activate to sort column ascending">{{ trans_choice('web::seat.location', 1) }}</th>
+                  <th class="sorting">Type</th>
+                  <th class="sorting">Content</th>
+                  <th class="sorting">
+                    {!! img('type', 4051, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
+                    Cargo Usage
+                    <!-- @todo - setup new words in language files -->
+                  </th>
+                  <th></th>
+                </tr>
+                </thead>
+                <tbody>
+
+
+                @foreach($starbases as $key => $starbase)
+
+                  @foreach($starbase->modules as $module)
+
+                  <tr>
+                    <td>
+                      {!! img('type', $module['detail']->typeID, 64, ['class' => 'img-circle eve-icon small-icon'], false) !!}
+                      {{ $module['detail']->typeName }}
+                      @if($module['detail']->typeName !== $module['detail']->itemName)
+                        <i>({{ $module['detail']->itemName }})</i>
+                      @endif
+                    </td>
+                    <td data-order="{{ $starbase->moonName }}">
+                      <b>{{ $starbase->moonName }}</b>
+                      <span class="
+                        @if($starbase->mapSecurity >= 0.5)
+                              text-green
+                            @elseif($starbase->mapSecurity < 0.5 && $starbase->mapSecurity > 0.0)
+                              text-warning
+                            @else
+                              text-red
+                            @endif">
+                        <i>({{ round($starbase->mapSecurity,  2) }})</i>
+                      </span>
+                    </td>
+                    <td>
+
+                      <span data-toggle="tooltip" title="" data-original-title="{{ $starbase->starbaseTypeName }}">
+                        {!! img('type', $starbase->starbaseTypeID, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
+                        {{ $starbase->starbaseName }}
+                      </span>
+                    </td>
+
+                    @if($module['used_volume'] == 0 || $module['available_volume'] == 0)
+                    <?php $usedvolume = "0%"; ?>
+                    @else
+                    <?php  $usedvolume = number(100 * ($module['used_volume']) / ($module['available_volume']), 0)."%" ?>
+                    @endif
```

Comment:
This block needs to be calculated in the Controller. There should be no `<?php` tags in templates.